src/libical/icaltz-util.c

```diff
@@ -38,6 +38,7 @@
 
 #if defined(__OpenBSD__) && !defined(bswap_32)
 #define bswap_32 swap32
+#define bswap_64 swap64
```

Comment:
would it be better to have:
```
#if defined(__OpenBSD__) && !defined(bswap_64)
#define bswap_64 swap64
#endif
```
?

Reply:
I don't think so. The check is just to determine which BSD is being used. Some use bswap macros and some use swap.